docs/overview/project.rst

```diff
@@ -0,0 +1,128 @@
+
+Due to the sensitive nature of research data and analysis information - especially in early stages of an analysis - accessibility of analysis information is subject to permissions set by the creator of an analysis and the researchers involved.
+
+When starting a new analysis, the creator of the analysis draft record will be the only one able to view and edit the record. He can grant read or edit rights to researchers in the analysis team or working group.
+
+As soon as the analysis is submitted, the analysis will be shared with the collaboration meaning its members will acquire rights to view the analysis. Editing rights will remain as they were for the draft.
+A draft version can be submitted any time. There is no drawback in keeping a draft until the analysis is published other than that analysis content is hidden from members of your collaboration.
+
+.. note::
+
+	- only CERN members have access to CAP login
+	- only collaboration members have access to a collaboration's area, can create analyses and can see shared analyses
+	- only a certain collaboration's members have access to this collaborations analyses
+	- only members granted specific rights can see or edit a draft version of an analysis
+	- only the creator can see or edit an analysis with default permission settings
+
+2. Search and Retrieve
+----------------------
+
+The search capability of CAP can help users find both archived and on-going analyses they have access to that contain the information specified in the search.
+
+Search capability
+~~~~~~~~~~~~~~~~~
+
+Using the search bar at the top of the page or the dedicated search page that comes with it, users can search through their own and all shared analyses within their collaboration, past or on-going. Based on feedback by collaboration physicists filters will be created to enable search for specific content.
+
+Additionally, users can easily keep track of and find analyses they created and those that are shared with them through a personal dashboard.
+
+All analysis metadata are indexed, which means users can find analyses with specific parameters, processed with a specific algorithm, or using a specific dataset or simulation to name a few examples. Information that is not explicitly added to the schema and instead stored in an uploaded file are not indexed for search right now.
+
+.. Facets/ filter
+.. ~~~~~~~~~~~~~~
+
+3. Review and compare
+---------------------
+
+CAP aims to simplify combining analysis as well as the process of analysis reviewing and approval by enabling the user to give specific access to analysis records and store relevant analysis information in one place.
+
+Report Compilation
+~~~~~~~~~~~~~~~~~~
+
+Analysis information will be exportable from CAP to generate a status report or a framework for the analysis note.
+CAP can also be a source of reference in analysis meetings.
```

Comment:
This either needs more context right now or should be postponed until further functionality is defined or integrated.